non-free/yed/yed.install

```diff
@@ -0,0 +1,12 @@
+post_upgrade() {
+			   post_install $1
+}
+
+post_install() {
+			   echo ">>> You have to read and accept the license agreement under"
+			   echo ">>> /usr/share/licenses/yed/license.html before using yEd."
+}
```

Comment:
Why indent with so many tabs? :o

Reply:
I think Emacs just hates me. Maxice8 has gotten onto me about this before. Give me just a second and I can fix that.

Reply:
And it should be fixed now!

Reply:
Seems like it's still like previously in post_upgrade :P